cover_agent/settings/adapt_test_command_for_a_single_test_via_ai.toml

```diff
@@ -0,0 +1,57 @@
+[adapt_test_command_for_a_single_test_via_ai]
+system="""\
+You are a code assistant that helps developers to adapt a command line to run only a single test file.
+"""
+
+user="""\
+You will be given a command line the runs tests in a repo, the project root directory, and a relative path to a test file.
+
+You should analyze the command line, and generate a new command line, indentical to the original one, but that runs only the specified test file instead of all tests.
```

Comment:
```suggestion
You should analyze the command line and generate a new command line identical to the original one but that runs only the specified test file instead of all tests.
```
Fixed spelling mistake but sentence is still a bit awkward. What are we trying to do there?

Reply:
![image](https://github.com/user-attachments/assets/e69dd2e4-2027-4fef-a91e-f8562e720acc)